test/unit/specs/components/common/__snapshots__/TmSessionImport.spec.js.snap

```diff
@@ -21,6 +21,480 @@ exports[`TmSessionImport has the expected html structure 1`] = `
+              <label
+                class="tm-form-group__label"
+                for="import-password-confirmation"
+              >
+                
+    Confirm Password
+  
+              </label>
+               
+              <div
+                class="tm-form-group__field"
+              >
+                <input
+                  class="tm-field"
+                  id="import-password-confirmation"
+                  placeholder="Enter password again"
+                  type="password"
+                />
+                 
+                <!---->
+              </div>
+            </div>
+             
+            <div
+              class="tm-form-group"
+            >
+              <!---->
+               
+              <label
+                class="tm-form-group__label"
+                for="import-seed"
+              >
+                
+    Seed Phrase
+  
+              </label>
+               
+              <div
+                class="tm-form-group__field"
+              >
+                <textarea
+                  class="tm-field-seed tm-field"
+                  id="import-seed"
+                  placeholder="Must be exactly 24 words"
+                  style="overflow-x: hidden; word-wrap: break-word; overflow-y: hidden;"
+                />
+                 
+                <!---->
+              </div>
+            </div>
+             
+            <div
+              class="tm-form-group"
+            >
+              <!---->
+               
+              <!---->
+               
+              <div
+                class="tm-form-group__field"
+              >
+                <div
+                  class="field-checkbox"
+                >
+                  <label
+                    class="field-checkbox-label"
+                    for="error-collection"
+                  >
+                    <input
+                      id="error-collection"
+                      type="checkbox"
+                    />
+                    
+              I'd like to opt in for remote error tracking to help improve
+              Voyager.
+            
+                  </label>
+                </div>
+              </div>
+            </div>
+          </div>
+           
+          <div
+            class="session-footer"
+          >
+            <button
+              class="tm-btn"
+            >
+              <span
+                class="tm-btn__container"
+              >
+                <!---->
+                <span
+                  class="tm-btn__value"
+                >
+                  Recover
+                </span>
+              </span>
+            </button>
+          </div>
+        </main>
+         
+        <!---->
+      </form>
+    </div>
+  </div>
+   
+  <button
+    class="tm-btn session-close"
+  >
+    <span
+      class="tm-btn__container tm-btn--secondary"
+    >
+      <!---->
+      <span
+        class="tm-btn__value"
+      >
+        Back to Lunie
```

Comment:
Back button is showing

Reply:
@faboweb That is not the back button I added but rather the close button from SessionFrame. It is not visible in the extension and has been there since sessionFrame was introduced.